Sygic.Corona.Profile/SendMfaToken.cs

```diff
@@ -36,7 +37,7 @@ public SendMfaToken(IMediator mediator, ValidationProcessor validation)
             {
                 string requestBody = await new StreamReader(req.Body).ReadToEndAsync();
                 var data = JsonConvert.DeserializeObject<SendMfaTokenRequest>(requestBody);
-                var command = new SendMfaTokenCommand(data.DeviceId, data.ProfileId);
+                var command = new SendMfaTokenCommand(data.DeviceId, data.ProfileId, Environment.GetEnvironmentVariable("MfaTokenMessage"));
```

Comment:
Pls discuss the case where no environment variable is there.
Add a default message in English out there.

Reply:
This is a mandatory message and it should be there in order for the system to work correctly.

Reply:
This is a mandatory message and it should be there for the system to work correctly.